### DISCUSSION TITLE
LPS firmware button issue (greyed out)

### DISCUSSION THREAD

**Discussion by michaelh85** [bitcraze/discussions#75] [Q&A]
Hello,
    Forgive me I am new and trying to learn the basics. I am having issues updating the firmware to the LPS nodes. I have tried a few different ways but all end up the same. I tried to connect with the LPS tools software on both Windows 11 and Ubuntu 22.04 but still had the greyed-out button. At one point I got the firmware button to activate but then I had an error code pop up. I will attach a picture of that error code below. I also get the same error code when trying to configure the node in the same lps configuration tool.

Thank you in advance for any help or ideas!

Cheers,
Mike
![IMG_3946 2](https://user-images.githubusercontent.com/105250661/171965370-f409b143-2a10-433a-86ec-1cfc3c62808f.JPG)

**Comment by krichardsson** [comment]
Hi Mike!
You probably have to fix the permissions for USB, see https://www.bitcraze.io/documentation/repository/crazyflie-lib-python/master/installation/usb_permissions/

**Reply by TopTalent333** [reply]
hi, I am his mate for this project.
I tried it already but no effect.
if you have experience with this, please let me know guideline for firmware updating on lps tools

**Reply by krichardsson** [reply]
Hi!
At the end of this forum thread https://forum.bitcraze.io/viewtopic.php?t=5375 there are some tips and tricks on how to debug USB on Ubuntu
